web_src/css/base.css

```diff
@@ -44,7 +44,7 @@ html, body {
 }
 
 body {
-  line-height: 1.4285rem;
+  line-height: 20px;
```

Comment:
Value is the same as before (which was 19.9999px), but I guess integer pixels will give the browser a better time at rendering and we have to prepare for eventually switching to 16px root font size.